client/app/src/main/java/com/microsoft/research/karya/ui/base/BaseActivity.kt

```diff
@@ -128,7 +130,7 @@ abstract class BaseActivity(
         }
 
         // Update UI
-        uiScope.launch {
+        runBlocking {
```

Comment:
Can you make this not suspending? We should be fetching the strings from resources which doesn't have to be suspending.

Reply:
Yeah, I will also remove the calls for getValueFromName since, we do not need them